### PR TITLE
Document quota caching flags

### PR DESCRIPTION
**Pull request by codingllama**

### CODE REVIEW

**Comment by codingllama** [comment]
Issue #490

File: quota/etcd/README.md [hunk 1]
```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
 1.1) is recommended, so there is some protection against token leakage without
 too much compromise of the quota system in exceptional situations.
 
+`--quota_max_cache_entries` and `--quota_min_batch_size` are related to token
+caching. Some level of token caching (i.e., both flags having values > 0) is
```
Comment (AlCutter) [inline quota/etcd/README.md]: "i.e.," -> "i.e."

File: quota/etcd/README.md [hunk 2]
```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
...
+recommended to lessen the latency impact of rate limiting.
+
+`--quota_min_batch_size` is the least number of tokens acquired from etcd. If a
```
Comment (AlCutter) [inline quota/etcd/README.md]: the least -> the minimum?

File: quota/etcd/README.md [hunk 3]
```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
...
+particular request demands less tokens than the minimal batch size, the
```
Comment (AlCutter) [inline quota/etcd/README.md]: less -> fewer

File: quota/etcd/README.md [hunk 4]
```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
...
+remaining tokens are kept in memory, saving new requests to etcd until those are
```
Comment (AlCutter) [inline quota/etcd/README.md]: kept -> cached

Comment (AlCutter) [inline quota/etcd/README.md]: potentially saving

File: quota/etcd/README.md [hunk 5]
```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
...
+insufficient.
```
Comment (AlCutter) [inline quota/etcd/README.md]: until those are consumed?

File: quota/etcd/README.md [hunk 6]
```diff
@@ -141,6 +145,20 @@ more tokens are restored per leaf batch. A value slightly higher than 1 (e.g.
...
+
+`--quota_max_cache_entries` relates to how many quota Specs are kept in the
```
Comment (AlCutter) [inline quota/etcd/README.md]: relates to -> determines?

File: quota/etcd/README.md (hunk 1 above)
Comment (codingllama) [reply quota/etcd/README.md]: Done.

File: quota/etcd/README.md (hunk 2 above)
Comment (codingllama) [reply quota/etcd/README.md]: Done.

File: quota/etcd/README.md (hunk 3 above)
Comment (codingllama) [reply quota/etcd/README.md]: Fewer

File: quota/etcd/README.md (hunk 4 above)
Comment (codingllama) [reply quota/etcd/README.md]: Done.

Comment (codingllama) [reply quota/etcd/README.md]: Done.

File: quota/etcd/README.md (hunk 5 above)
Comment (codingllama) [reply quota/etcd/README.md]: Ta. I struggled to find a better word here.

File: quota/etcd/README.md (hunk 6 above)
Comment (codingllama) [reply quota/etcd/README.md]: Done.

**Comment by codingllama** [comment]
PTAL?

**Comment by codingllama** [comment]
Friendly ping?

**Comment by codingllama** [comment]
Friendly ping?

**Pull request merged**